crates/oxc_linter/src/rules/react/jsx_key.rs

```diff
@@ -400,6 +444,29 @@ fn test() {
           }
         ];
         ",
+        r"{React.Children.toArray(items.map((item) => {
+            return (
+              <div>
+             {item}
+             </div>
+              );}))}
+        ",
+        r#"import { Children } from "react";        
+        Children.toArray([1, 2 ,3].map(x => <App />));
+        "#,
+        r"
+        import Act from 'react';
+        import { Children as ReactChildren } from 'react';
+
+        const { Children } = Act;
+        const { toArray } = Children;
+
+        Act.Children.toArray([1, 2 ,3].map(x => <App />));
+        Act.Children.toArray(Array.from([1, 2 ,3], x => <App />));
+        Children.toArray([1, 2 ,3].map(x => <App />));
+        Children.toArray(Array.from([1, 2 ,3], x => <App />));
+        ",
+        r"React.Children.toArray([1, 2 ,3].map(x => <App />));",
     ];
 
     let fail = vec![
```

Comment:
Could you add a test case `foo.Children.toArray([1, 2 ,3].map(x => <App />));'`? This should fail.